src/Hive.IO/Hive.IO/Results/Results.cs

```diff
@@ -505,7 +505,10 @@ public static List<double[]> GetMonthlySolarGainsPerWindow(Building.Building bui
                 for (int j = 0; j < zone.Windows.Length; j++)
                 {
                     var solarGains = new double[Misc.MonthsPerYear];
-                    zone.MonthlySolarGainsPerWindow[j].CopyTo(solarGains, 0);
+                    if (zone.SolarGainsPerWindowMonthly[j].Length == Misc.HoursPerYear)
+                        Misc.GetCumulativeMonthlyValue(zone.SolarGainsPerWindowMonthly[j]).CopyTo(solarGains, 0);
```

Comment:
shouldn't this be `zone.SolarGainsPerWindow**Hourly**[j]` ?

Reply:
good catch. shouldn't be monthly, but we can read it directly from zone.SolarGainsPerWindowMonthly

adressed in d0a6533